elephant/cubic.py

```diff
@@ -0,0 +1,212 @@
+'''
+CuBIC is a statistical method for the detection of higher order of
+correlation in of parallel spike train based on the analysis of the
+cumulants of the population count.
+Given a list sts of SpikeTrains, the analysis comprises the following
+steps:
+1) compute the population histogram (PSTH) with the desired bin size
+   >>> binsize = 5 * pq.ms
+   >>> pop_count = elephant.statistics.time_histogram(sts, binsize)
+2) apply CuBIC to the population count
+   >>> aplha = 0.05  # significance level of the tests used
+   >>> xi, p_val, k = cubic(data, ximax=100, alpha=0.05,\
+           errorval=4.):
+'''
+# -*- coding: utf-8 -*-
+from __future__ import division
+import scipy.stats
+import scipy.special
+import math
+import warnings
+
+
+# Based on matlab code by Benjamin Staude
```

Comment:
@apdavison  @pietroquaglio 
Sonja contacted B.Staude. 
He is happy to see that his work is still under further development and willingly agrees to integrate Cubic in Elephant.